src/components/FooterModule.vue

```diff
@@ -96,20 +96,20 @@
         <router-link to="/Legal/TermsOfUse">terms of use</router-link>. CVE is sponsored by the
         <a href="https://www.dhs.gov/">U.S. Department of Homeland Security (DHS)
           <span class="icon cve-icon-xxs">
-            <p id="extenalLink5" class="is-hidden">external link</p>
```

Comment:
These ids are unlikely to be externally referenced, but, as w/ everything else, dropping them isn't a big deal.